modules/ROOT/pages/upgrade-index.adoc

```diff
@@ -1,35 +1,282 @@
+Run the following command:
+```
+/opt/anypoint/runtimefabric/rtfctl version
+```
+
+Output includes:
+
+* The `rtfctl` component version in the `rtfctl` field
+* The installer package component version in the `appliance` field
+* The Runtime Fabric component version in the `agent` field
 
-If an appliance upgrade is available, a code snippet for performing the upgrade is provided, as shown in the following example:
+For example:
 ```
-rtfctl appliance upgrade --url https://<upgrade-package-information>
+[root@ip-10-0-245-74 /]# /opt/anypoint/runtimefabric/rtfctl version
+COMPONENT    VERSION                  
+rtfctl       0.2.96+2868e14           
+agent        1.5.4                    
+appliance    1.1.1581641679-5884bce   
+kubernetes   1.13.12
 ```
 
+[NOTE]
+The Runtime Fabric component and cluster-level component (appliance) versions that are in use are also displayed in the *Runtime Fabrics* tab in Runtime Manager. If an upgrade is available, an "Upgrade to vx.x" message is displayed.
+
+== Determine the Most Recent Component Versions Available
+The most recent component versions are displayed in the *Downloads* page, located in Runtime Manager in the *Runtime Fabrics* tab. A link to each package’s release notes is provided to show the changes.
+
+You can also review the xref:release-notes::runtime-fabric/runtime-fabric-release-notes.adoc[Anypoint Runtime Fabric Release Notes] information.
+
+=== Apply the Upgrade
+Before upgrading your production environment, perform the following steps on a nonproduction Runtime Fabric to verify compatibility with your environment.
+
+. Review the xref:release-notes::runtime-fabric/runtime-fabric-release-notes.adoc[Anypoint Runtime Fabric Release Notes] information.
+. Provision additional worker VMs as needed to maintain availability of applications:
+
+.. If the amount of available resources for application deployments is less than the resources of one worker node, add another worker VM with the same amount of CPU, memory, and disk resources to the Runtime Fabric.
++
+See xref:manage-nodes.adoc[How to Add a Node to Runtime Fabric].
+. To upgrade Runtime Fabric components to a later version:
+
+.. Plan to upgrade when your team is not deploying new applications or managing existing applications on Runtime Fabric.  
+.. Navigate to your Runtime Fabric by selecting it on the *Runtime Fabrics* tab in Runtime Manager.
+.. Select "Upgrade to vx.x". The status of the Runtime Fabric is updated to show that the upgrade procedure is in progress.
+.. Wait for the status to transition to *Active*. 
++
+In most cases, the upgrade takes less than 10 minutes. When the status transitions to *Active*, and the version of the Runtime Fabric reflects the new version, the upgrade is complete.
++
+If the Runtime Fabric and Runtime Manager connection disconnects during the upgrade, connectivity is restored after the upgrade finishes.
++
+Keep your browser session open to use in later steps.
```

Comment:
Should this be the final step of the task preceding it?

Reply:
Or...should this be a step at all, given that "later steps" are provided next?
Please review narrative flow here.